server/lib/routes/get-index.js

```diff
@@ -9,9 +24,40 @@ module.exports = function () {
   route.path = '/';
 
   route.process = function (req, res) {
-    res.render('index');
+    if (! shouldEmitActivityEvent()) {
+      return res.render('index');
+    }
+
+    var time = Date.now();
+    var flowId = crypto.randomBytes(16).toString('hex');
+    res.render('index', { flowBeginTime: time, flowId: flowId });
+
+    setImmediate(function () {
+      // This event data is structured to be consistent with the auth server
+      // activity events. So yes, the format is different to the content server
+      // log lines. And yes, there is a mix of camelCase and under_score keys.
+      var activityEvent = {
+        device_id: null, //eslint-disable-line camelcase
+        event: 'flow.begin',
+        flow_id: flowId, //eslint-disable-line camelcase
+        flow_time: 0, //eslint-disable-line camelcase
+        new_user: false, //eslint-disable-line camelcase
+        time: time,
+        uid: null,
+        userAgent: req.headers['user-agent']
```

Comment:
Just realised this should probably be parsed before emitting the event. I was assuming that it would be done at the ingestion point before but that seems unlikely in light of https://github.com/mozilla/fxa-content-server/issues/3551#issuecomment-193080411. Will update the PR.